app.js

```diff
@@ -0,0 +1,66 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        // title: "",
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+})
+
+app.Views.TodoApp = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        this.newTodoField = this.$(".todo__new-todo-field");
+        this.markAllCompleteField = this.$(".mark-all-complete");
+        this.totalCount = this.$(".total-count");
+        this.remainingCount = this.$(".remaining-count");
+        this.completeCount = this.$(".complete-count");
+    }
+});
+app.Views.TodoItem = Backbone.View.extend({
+    tagName: "li",
+    className: "todo-item",
+    template: Handlebars.compile($("#todo-item-template").html()),
```

Comment:
Should setup your template in your initialize

Reply:
If I put it in initialize, won't Handlebars do a repeated compilation of the template every time a `TodoItemView` is constructed and initialized?

Reply:
I am pretty sure Musharraf is correct here. What is the reasoning we do it in initialize? was it part of how we were doing JIT prior to your refactor?